docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -133,10 +133,10 @@ Mapping examples:
 | service-port           | SERVICE_PORT             | Service:<br>--Port:                             |
 | clients-core-data-host  | CLIENTS_CORE_DATA_HOST  | Clients:<br>--core-data:<br>----Host:   |
 | edgex-startup-duration | [EDGEX_STARTUP_DURATION] | -                                                   |
-| add-secretstore-tokens | [ADD_SECRETSTORE_TOKENS] | -                                                   |
+| add-secretstore-tokens | [EDGEX_ADD_SECRETSTORE_TOKENS] | -                                                   |
```

Comment:
```suggestion
| edgex-add-secretstore-tokens | [EDGEX_ADD_SECRETSTORE_TOKENS] | -                                                   |
```

Reply:
Good catch. Updated in this commit. https://github.com/edgexfoundry/edgex-docs/pull/1027/commits/aedb72fbed5548c60ee29fe1cb133a201045ef9a